src/edu/ufl/jjr/writingLog/WritingLog.java

```diff
@@ -0,0 +1,99 @@
+package edu.ufl.jjr.writingLog;
+
+import edu.ufl.jjr.peer.Peer;
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.logging.Logger;
+import java.util.logging.FileHandler;
+import java.util.logging.SimpleFormatter;
+
+// https://docs.oracle.com/javase/7/docs/api/java/util/logging/Logger.html
+// https://www.codota.com/code/java/classes/java.util.logging.FileHandler
+// https://github.com/CNT4007-JJR/P2P-project/blob/3-logging-class/src/edu/ufl/jjr/writingLog/WritingLog.java
+// https://stackabuse.com/how-to-get-current-date-and-time-in-java/
+
+public class WritingLog {
+
+    private Peer peer;
+    private int peerID;
+    private Logger logger;
+    private FileHandler fh;
+
+    public WritingLog(Peer peer) {
+        this.peer = peer;
+        this.peerID = peer.peerID;
+        this.logger = Logger.getLogger(Integer.toString(peerID));
+
+        // this path needs to be abstracted, right now it is hard coded
+        String dir = "/Users/jonathanmorales/Desktop/P2P-project/project/log_peer_" + Integer.toString(peerID) + ".log";
```

Comment:
I believe setting the dir to "log_peer_" + PeerId ".log"  should write to the directory of the executable which is the desired functionality